src/components/Modals/EditThumbnailModal/EditThumbnailModal.tsx

```diff
@@ -61,7 +61,7 @@ export default class EditThumbnailModal extends React.PureComponent<Props, State
 
     let wearablePreviewProp
     if (file) {
-      wearablePreviewProp = { blob: toWearableWithBlobs({ file, isEmote: true }) }
+      wearablePreviewProp = { blob: toEmoteWithBlobs({ file }) }
```

Comment:
we should add some logic here to know if it should use `toEmoteWithBlobs` or `toWearableWithBlobs`, right?

Reply:
The EditThumbnailModal is just for emotes I think (it is the one with the slider to pick the frame), so no support for WearableWithBlobs

Reply:
Yes, correct! perfect then 👍